client/src/app/slot-fill/__tests__/SlotFillSpec.js

```diff
@@ -210,6 +211,81 @@ describe('slot-fill', function() {
       expect(slot.contains(fill)).to.be.true;
     });
 
+
+    describe('ordering', function() {
+
+      it('should display fills ordered alphabetically by group', function() {
+
+        // given
+        var unorderedFills = [ '1_a', '2_b', '3_a', 'foo', '2_a' ].map(id => (
+          <Fill name="foo" group={ id } key={ id }>
+            <div className="fill" id={ id } />
+          </Fill>
+        ));
+
+        // when
+        var slotFillRoot = ReactDOM.render(
+          <SlotFillRoot>
+            { unorderedFills }
+            <div className="slot">
+              <Slot name="foo" />
+            </div>
+          </SlotFillRoot>,
+          container
+        );
+
+        // then
+        var fills = findComponentsWithClass(slotFillRoot, 'fill'),
+            slot = findRenderedDOMComponentWithClass(slotFillRoot, 'slot');
+
+        expect(fills.every(fill => slot.contains(fill))).to.be.true;
+        expect(fills.map(fill => fill.id)).to.eql([
+          '1_a',
+          '2_a',
+          '2_b',
+          '3_a',
+          'foo'
+        ]);
+
+      });
+
+
+      it('should display fills ordered by priority inside same group', function() {
+
+        // when
+        var slotFillRoot = ReactDOM.render(
+          <SlotFillRoot>
+            <Fill name="foo" group="1_a" priority={ -1 }>
```

Comment:
As we are reworking the way fills work, does it make sense to rename `Fill#name` to `Fill#slot`? Looking at the code nowadays I find the `name` property very unintuitive.

Reply:
Sounds reasonable. `name` seems to suggest that it's a fill's name when in fact it is a slot's name. I'll have another commit for that.

Reply:
Done.

Reply:
Much better now, thanks for the additional update.